.github/workflows/build-ffi.yml

```diff
@@ -1,6 +1,22 @@
 name: Pact-Rust FFI Build
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
I'll remove that, that would be annoying. Imagine waiting for a build, then someone else pushes a commit.

Reply:
The concurrency group is per-PR

Reply:
So there should not be a case of waiting for a build, and it being canceled by someone else's push _except_ if you are both working on the same PR.

If you want this to run specifically when creating tagged released, I recommend creating a separate trigger for tags.